### DISCUSSION TITLE
/metrics exposed

### DISCUSSION THREAD

**Discussion by roughnecks** [ente-io/ente#2562] [Q&A] [answered]
Hello,

I just found out that my nginx reverse proxy for ente-museum was exposing the `/metrics` endpoint, so I just added the following to virtualhost:

```
        location /metrics {
                allow ::1;
                deny all;
                return 404;
        }
```

does it pose a security risk?

**Comment by mnvr** [comment, marked as answer]
Removed :8080/metrics in https://github.com/ente-io/ente/pull/2569 (that PR has more details)
